Docker/input/oval/installed_app_is_docker.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="inventory" id="installed_app_is_docker" version="1">
+    <metadata>
+      <title>Docker</title>
+      <affected family="unix">
+        <product>Docker</product>
+      </affected>
+      <reference ref_id="cpe:/a:docker:docker" source="CPE" />
```

Comment:
Should probably add ` cpe:/a:redhat:docker` as well since it is defined in the NIST CPE database.